layers/+tools/lsp/funcs.el

```diff
@@ -43,7 +43,13 @@ https://github.com/emacs-lsp/lsp-javascript/issues/9#issuecomment-379515379"
   (interactive)
   (defun lsp-prefix-company-transformer (candidates)
     (let ((completion-ignore-case t))
-      (all-completions (company-grab-symbol) candidates)))
+      (if (and (car candidates)
+               (get-text-property 0 'lsp-completion-prefix (car candidates)))
+          (all-completions (company-grab-symbol) candidates)
+        candidates
+        )
+      )
+    )
```

Comment:
All these closing parenthesis must be at the end of line 49 as this is the style for writing lisp code.